### DISCUSSION TITLE
Slider circles don't go away when you slider break.

### DISCUSSION THREAD

**Discussion by PolyCatDev** [ppy/osu#19834] [Q&A]
### Type

Cosmetic

### Bug description

Slider circles don't go away. This issue is annoying only in beatmaps like Mayday where there are a lot of overlapping sliders. This creates some sort of an explosion.

### Screenshots or videos
![LazerSkinOsu!Lazer](https://user-images.githubusercontent.com/88086950/185354033-cd39bb53-9618-4caa-8330-bc1b1fdeabfa.GIF)

![Screenshot from 2022-08-17 20-43-01](https://user-images.githubusercontent.com/88086950/185207293-7667b841-648d-48c8-93f1-c73720791980.png)

### Version

2022.816.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9365077/database.log)
[network.log](https://github.com/ppy/osu/files/9365078/network.log)
[performance.log](https://github.com/ppy/osu/files/9365080/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9365081/runtime.log)

**Comment by peppy** [comment]
Does it look better to you with the classic skin enabled?

**Comment by PolyCatDev** [comment]
Not really. There are a lot of balls.
![ClasicSkinOsu!Lazer](https://user-images.githubusercontent.com/88086950/185352438-90377280-d798-419c-b519-1f480348b536.GIF)

And this is how it looks with my skin (an edited version of the GRIS project: https://skins.osuck.net/index.php?newsid=1285)
![MySkinOsu!Lazer](https://user-images.githubusercontent.com/88086950/185355416-8fe99df1-021c-4491-b956-6796900941bf.GIF)


btw. I think this might be because of the new(ish) slider end animation.

**Comment by peppy** [comment]
Hmm, I think it's only by chance that stable doesn't do this.

The new behaviour is *more correct* as far as game mechanics go... will wait for more opinions on how this should look.

**Reply by PolyCatDev** [reply]
I think a switch to turn off the behavior to the classic mod customization settings would do the trick.

**Reply by crumpledmint** [reply]
I don't know what's intended behaviour but on stable you can catch last sliderb to avoid sliderbreaks on the other sliders that were played simultaneously with the last slider so it may also affect gameplay on aspire maps

**Reply by PolyCatDev** [reply]
In older versions of lazer if you sliderbroke that was it, the circle wasn't coming back. I don't have a problem with the new behavior, I actually really like it in normal beatmaps. This only seems to affect aspire maps, and the only thing I can think of is an Aspire mod or a customization setting to stop the effect visually in the classic mod.
